### PR TITLE
216.1 KK CDR processing/rescheduling

### PR DESCRIPTION
**Pull request by frankhuster**
Trying a new pull request to see if that fixes CI build problems

### CODE REVIEW

**Comment by larubbio** [comment]
ok to test

**Comment by koshalt** [comment]
ok to test

File: imi/pom.xml [hunk 1]
```diff
@@ -56,6 +56,13 @@
             <version>${motech.version}</version>
         </dependency>
 
+        <!-- Event relay -->
+        <dependency>
+            <groupId>org.motechproject</groupId>
+            <artifactId>motech-platform-event</artifactId>
```
Comment (larubbio) [inline imi/pom.xml]: It just dawned on me how little we are using the event system.

Comment (llavoie) [inline imi/pom.xml]: Rather surprising, no?

Comment (frankhuster) [inline imi/pom.xml]: Get off my lawn!

Comment (koshalt) [inline imi/pom.xml]: hiii

Comment (frankhuster) [inline imi/pom.xml]: waaa

**Pull request merged**